--- rules.bzl
+++ rules.bzl
@@ -30,6 +30,10 @@ def _android_ndk_repository_impl(ctx):
     if ndk_path.startswith("$WORKSPACE_ROOT"):
         ndk_path = str(ctx.workspace_root) + ndk_path.removeprefix("$WORKSPACE_ROOT")
 
+    if not ctx.path(ndk_path).exists:
+        ctx.file("BUILD.bazel", "")
+        return
+
     is_windows = False
     executable_extension = ""
     if ctx.os.name == "linux":
